pkg/datapath/loader/base.go

```diff
@@ -372,8 +374,24 @@ func (l *Loader) Reinitialize(ctx context.Context, o datapath.BaseProgramOwner,
 		// Enable IPv6 for now
 		sysSettings = append(sysSettings,
 			sysctl.Setting{Name: "net.ipv6.conf.all.disable_ipv6", Val: "0", IgnoreErr: false})
+
+		localIPv6CIDR := node.GetIPv6AllocRange()
+		fmt.Printf(">>>>> cidr %s\n", localIPv6CIDR.String())
+		addrs, err := link.GetIPv6Addresses(hostDev1.Attrs().Name)
+		if err != nil {
+			return err
+		}
+		args[initArgStaleIPv6NodeIP] = "<nil>"
+		for _, addr := range addrs {
+			fmt.Printf(">>>>> matching: %s\n", addr.IPNet.IP.String())
+			if localIPv6CIDR.Contains(addr.IPNet.IP) {
+				args[initArgStaleIPv6NodeIP] = addr.IPNet.IP.String()
+				break
+			}
+		}
```

Comment:
Or we delete the stale ipv6 here, when we generate the new ipv6 for cilium_host.

Which one do you prefer? For me, deleting stale ipv6 at init.sh seems to be more cohesive, as that will be the only place we change the cilium_host's ipv6, but we have to pass an additional shell argument.

(There are other options, like deleting when installing ipv6 routes, but that's way too in-cohesive, so I ditched those ideas.)

Reply:
As we are migrating from `bpf/init.sh`, I'd suggest to delete it here.